assets/javascripts/kitten/karl/pages/modal-edit-project/index.js

```diff
@@ -0,0 +1,271 @@
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    @media ${mq.mobile} {
+      min-height: ${pxToRem(285)};
+    }
+
+    :hover,
+    :focus {
+      cursor: pointer;
+
+      .k-ModalEditProject--button {
+        border-color: ${COLORS.primary2};
+        background-color: ${COLORS.primary2};
+      }
+    }
+  }
+
+  .k-ModalEditProject--paragraph {
+    flex: 1;
+    margin-bottom: ${pxToRem(20)}!important;
+    margin-top: 0;
+  }
+
+  .k-ModalEditProject--blockText {
+    display: flex;
+    align-items: baseline;
+    margin-top: ${pxToRem(10)};
+    line-height: normal;
+  }
+`
+
+const ModalEditProject = () => {
+  return (
+    <>
+      <GlobalStyle />
+      <Modal
+        size="big"
+        trigger={null}
+        isOpen
+        hasCloseButton
+        className="k-ModalEditProject--modal"
+      >
+        {() => (
+          <>
+            <Modal.Block>
+              <Modal.Title>
+                <Title
+                  tag="h1"
+                  modifier="quaternary"
+                  className={classNames(
+                    'k-u-align-left@m-up',
+                    'k-u-align-center@s-down',
+                    'k-u-margin-top-quadrupleHalf',
+                    'k-u-margin-bottom-tripleHalf',
+                  )}
+                >
+                  Créer un nouveau projet
+                </Title>
+              </Modal.Title>
+            </Modal.Block>
+            <Modal.Block>
+              <Grid>
+                <GridCol col-s="6">
+                  <button
+                    type="button"
+                    href="#"
+                    className={classNames(
+                      'k-u-margin-right-noneHalf@m-up',
+                      'k-u-margin-bottom-double@s-down',  
+                    )}
+                  >
+                    <Modal.Paragraph className="k-ModalEditProject--paragraph">
+                      <InfiniteIconNext width="47" height="47" color={COLORS.primary1} />
+                      <Title
+                        tag="p"
+                        modifier="septenary"
+                        className="k-u-margin-top-none k-u-margin-bottom-double"
+                      >
+                        Don permanent
+                      </Title>
+                      <div className="k-u-align-left">
+                        <Text
+                          tag="p"
+                          size="tiny"
+                          color="font1"
+                          lineHeight="normal"
+                          className="k-u-margin-none"
+                        >
+                          Avoir une page de don permanent vous permet de récolter des 
+                          fonds sur la durée grâce au système de&nbsp;:
+                        </Text>
+                        <div className="k-ModalEditProject--blockText">
+                          <CheckedIcon color={COLORS.primary1} width="12" height="12" />
```

Comment:
On peut rajouter des `aria-hidden` sur toutes les icônes pour être certains qu'elles ne gêneront pas les lecteurs d'écrans, elles sont toutes illustratives et ne portent pas de sens

```suggestion
                          <CheckedIcon color={COLORS.primary1} width="12" height="12" />
```